python/dgl/distributed/fake_kvstore.py

```diff
@@ -0,0 +1,58 @@
+"""Define a fake kvstore
+
+This kvstore is used when running in the standalone mode
+"""
+
+from .. import backend as F
+
+class KVClient(object):
```

Comment:
Can we put the logic into the real kvstore? It is a little weird we have a `fake_kvstore.py` here.

Reply:
it can. but i don't want to make the logic of kvstore too complex. maybe you don't like the name "fake". I think we can rename it as standalone_kvstore.py